### DISCUSSION TITLE
Make raid (and maybe dungeon as well) IDs like World Bosses locks

### DISCUSSION THREAD

**Discussion by vvranceanu** [chromiecraft/chromiecraft#6558] [Suggestions]
Would be cool if we could ran a raid multiple times per week to help guildies(without the need of alts), without loots even, similar to how world bosses work

**Comment by balleny** [comment]
i kinda like the concept of the character-bound ids. you can kill a boss multiple times but receive loot just once per week.
unfortunately, i expect the effort for implementation and ironing out all the abuse possibilities to be quite complex

**Comment by krutoj87** [comment]
One way to abuse this is offering sell runs. After the guild has cleared the raid, they sell a clear run to one person that has no competition on any of the items. And they could repeat this as often as they want. 

I don't know if sell runs would be a problem on CC, but with a system like this, the possibility would exist.
